roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -98,5 +99,6 @@
       owner: root
       group: root
   when:
-  - not ca_bundle_result
+  - ca_bundle_result.results[0].rc != 0
```

Comment:
You need to switch the order of the two first two conditionals.  Ansible should check if the item is defined first before checking the content.

/lgtm cancel